index.html

```diff
@@ -1,12 +1,50 @@
 <!DOCTYPE html>
 <html lang="en-us">
-  
+
 <head>
-  <meta charset ="utf-8">
+  <meta charset="UTF-8">
```

Comment:
utf-8 is the the default with html5, so not really necessary. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#attr-charset